File: csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/BadEventQueueIdErrorAllOf.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class BadEventQueueIdErrorAllOf {
    /// <summary>
    /// Gets or Sets Result
    /// </summary>
    [DataMember(Name="result", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "result")]
    public Object Result { get; set; }

    /// <summary>
    /// Gets or Sets Msg
    /// </summary>
    [DataMember(Name="msg", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "msg")]
    public Object Msg { get; set; }

    /// <summary>
    /// Gets or Sets Code
    /// </summary>
    [DataMember(Name="code", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "code")]
    public Object Code { get; set; }

    /// <summary>
    /// The string that identifies the invalid event queue. 
    /// </summary>
    /// <value>The string that identifies the invalid event queue. </value>
    [DataMember(Name="queue_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "queue_id")]
    public string QueueId { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class BadEventQueueIdErrorAllOf {\n");
      sb.Append("  Result: ").Append(Result).Append("\n");
      sb.Append("  Msg: ").Append(Msg).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  QueueId: ").Append(QueueId).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return Newtonsoft.Json.JsonConvert.SerializeObject(this, Newtonsoft.Json.Formatting.Indented);
    }

}
}

File: dart-jaguar/lib/model/bad_event_queue_id_error_all_of.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'bad_event_queue_id_error_all_of.jser.dart';

class BadEventQueueIdErrorAllOf {
  
  @Alias('result', isNullable: true,  )
  final Object result;
  
  @Alias('msg', isNullable: true,  )
  final Object msg;
  
  @Alias('code', isNullable: true,  )
  final Object code;
   /* The string that identifies the invalid event queue.  */
  @Alias('queue_id', isNullable: false,  )
  final String queueId;
  

  BadEventQueueIdErrorAllOf(
      

{
     this.result = null,  
     this.msg = null,  
     this.code = null,  
     this.queueId = null 
    
    }
  );

  @override
  String toString() {
    return 'BadEventQueueIdErrorAllOf[result=$result, msg=$msg, code=$code, queueId=$queueId, ]';
  }
}

@GenSerializer(nullableFields: true)
class BadEventQueueIdErrorAllOfSerializer extends Serializer<BadEventQueueIdErrorAllOf> with _$BadEventQueueIdErrorAllOfSerializer {

}

